#105679 release notes (#595)
valtimo-platform/valtimo-documentation: release-notes/major12/12.0.0/valtimo-backend-libraries.md

  If the plugin configuration file contained a 'null' value for a plugin property, an error was thrown.

* **Process not linked after deploying a new case definition**

  When a new version of the case definition was deployed, the new case definition would sometimes no longer be linked to

>   any process definitions. Even though the link existed in previous versions.

Reviewer: ```suggestion
  any of the process definitions that were linked in previous versions.
```
